packages/renderer/src/lib/ui/Select.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+export let id: string | undefined = undefined;
+export let name: string | undefined = undefined;
+export let value: unknown;
+export let readonly: boolean = false;
+export let disabled: boolean = false;
+export let required: boolean = false;
+export let error: boolean = false;
+
+let enabled: boolean = true;
+$: enabled = !readonly && !disabled;
+</script>
+
+<select
+  class="w-full p-1 outline-none bg-transparent border-[1px] border-transparent text-sm"
+  class:border-b-[var(--pd-input-field-stroke)]="{enabled && !error}"
+  class:border-b-[var(--pd-input-field-stroke-error)]="{enabled && error}"
+  class:border-b-[var(--pd-input-field-stroke-readonly)]="{!enabled}"
+  class:text-[color:var(--pd-input-field-focused-text)]="{enabled}"
+  class:text-[color:var(--pd-input-field-disabled-text)]="{!enabled}"
+  class:hover:border-[var(--pd-input-field-stroke)]="{enabled && !error}"
+  class:hover:border-[var(--pd-input-field-stroke-error)]="{enabled && error}"
+  class:hover:bg-[var(--pd-input-field-hover-bg)]="{enabled}"
```

Comment:
I would not grab the colors of the input component. I think it'll make us problems later

Of course the final color might be the same but I think people could be to override specific colors

Reply:
After thinking about it, yes, you're right. I am worried that if there are (e.g.) 17 different variables to change to make purple into blue, nobody will do it. I'll follow up via the UX meeting.

Reply:
Switching to using dropdown variables - identical pattern to Input, but having it's own colors.